_posts/2017-09-28-our-engineering-culture-methodologies-and-philosophies.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title:  "Our set of engineering culture and values"
+date:   2016-09-30 14:12:09 -0400
+categories: engineering culture
+---
+
+Every engineering group is different. Every engineering manager values different
+aspects of its team. This list was put together by the engineering team at
+_Forward Financing_.  
+
+## Engineer's Flow  
+There is this engineering management theory called "Flow". It's a state any
+engineer achieves to get into that is void of interruption and disruption. This
+is when engineers do their best work, can tackle the hardest problems, and why
+you'll often find our room either full of energy and boisterous or very very
+quiet to the point you can hear a pin drop. We do many things to help get
+engineers into Flow, some rules we follow:  
+1) Do not disturb if headset is on.
+2) Turn off slack notifications when working on larger/harder problems.
+3) Flexible work from home policy to reduce visual noise and disruption from
+outside parties.
+
+Here are some articles that speak specifically about flow or topics close in
+nature to flow.
+- http://www.paulgraham.com/makersschedule.html
+- http://firstround.com/review/track-and-facilitate-your-engineers-flow-states-in-this-simple-way/
+- https://hackernoon.com/its-a-wonder-some-teams-get-anything-done-64104f59bf5e
+  
+## Scrum  
+Scrum is an Agile methodology that help teams get things done in as productive
+a manner as possible. It allows the business to track project progress
+transparently and get an idea of throughput based on estimation. Scrum allows
+the product team (Product Managers and Engineer) to be able to set expectations
+that stakeholders understand and abide by. While none of us believe in burn down
+charts, for those people that are far removed from the process, it can act as
+a simple flag as to a sprint going well, or falling behind. The most important
+piece of Scrum is the Retrospective. This is an open forum to discuss what went
+well during the sprint, what failed, and what we can do as a team to address
+the issues that led to the failures. Thus far, we have evolved our process with
+our current team to address the issues that have arisen so far, and it has all
+been for the better. You can find our retrospective notes in Confluence.
```

Comment:
This blog post is public facing. Rephrase to say that something like "We archive our retrospective notes in Confluence", unless we plan on releasing those notes to the public.